### ISSUE TITLE
Strategy submission

### ISSUE THREAD

**Issue by vaughnmck** [butterdao/whip#76]
# Problem

Whip is currently restricted to strategies that the developers select, e.g. diversify into ETH or USDC. It is not possible to backtest a token or strategy selected by the user.

This is by design. We believe that Whip's utility is derived from providing a curated list of tokens/strategies that address common Treasury risks and exposures, not the backtesting infrastructure alone. We also attempt to solve an expertise and discovery problem, in that the user is not required to know which strategy/token can address their issue or search the large number of tokens and strategies available.

# Solution

Users can submit strategies for inclusion using a form with the following fields:
- Strategy Description
- Token Contract Address(es)
- Email address
- Other

**Comment by vaughnmck** [comment]
@lajarre this was your suggestion so I'd love your thoughts

**Comment by lajarre** [comment]
Let's launch this form right away so as to learn which kind of strategies will Whip need to support, and confirm interest in this feature.

In parallel, let's try to do some research: what other piece of software is doing the same kind of job? How automatized are they? Can we get any info on how?

**Comment by vaughnmck** [comment]
Agreed. For the sake of simplicity, let's start with a simple button that pops a modal form. We can use typeform if you want to avoid building the build and data management:
![Add_Strategy](https://user-images.githubusercontent.com/7321643/178743768-ed042774-3d34-4682-ba43-9417db43a6de.png)

On the research front, I have no knowledge of anything in crypto that does exactly the same thing but the user-submitted strategies model [pulls straight](https://docs.yearn.finance/developers/v2/getting-started) [from Yearn](https://github.com/yearn/brownie-strategy-mix) and has been copied [by a](https://docs.rari.capital/nova/) [few projects](https://docs.babylon.finance/developers/building-on-babylon#what-can-you-build).
